udata/templates/dataset/display.html

```diff
@@ -148,13 +148,13 @@ <h4 class="list-group-item-heading">
                             {{ share_btn(dataset.title) }}
                         </div>
                         <div class="btn-group btn-group-sm">
-                            {{ integrate_btn(dataset.id) }}
+                            {{ integrate_btn(dataset) }}
                         </div>
                         <div class="btn-group btn-group-sm">
                             {{ issues_btn(dataset) }}
                         </div>
                     </div>
-                </div>
+                </div></div>
```

Comment:
No, I wanted to limit the diff, `row` and `col-` divs are open on the same line, so I close it on the same line too. Otherwise, I had to increment the entire block each time => more diff lines